<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Achievements</class>
 <widget class="QDialog" name="Achievements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QLabel" name="__achievements">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;成 就&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="__ach_1">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>560</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_9">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_10">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_11">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_1_12">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹新手(I)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹新手(II)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹新手(III)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹专家(I)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹专家(II)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹专家(II)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹大师(I)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹大师(II)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;爆弹大师(III)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_10">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>270</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;YUAN***E(I)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_11">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>370</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;YUAN***E(II)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_3_12">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>470</y>
     <width>200</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;YUAN***E(III)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_9">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_10">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_11">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>370</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="__ach_2_12">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/a-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>__achievements</zorder>
  <zorder>__ach_1</zorder>
  <zorder>__ach_1_4</zorder>
  <zorder>__ach_1_6</zorder>
  <zorder>__ach_1_7</zorder>
  <zorder>__ach_1_8</zorder>
  <zorder>__ach_1_9</zorder>
  <zorder>__ach_1_11</zorder>
  <zorder>__ach_1_12</zorder>
  <zorder>__ach_3_1</zorder>
  <zorder>__ach_3_2</zorder>
  <zorder>__ach_3_3</zorder>
  <zorder>__ach_3_4</zorder>
  <zorder>__ach_3_5</zorder>
  <zorder>__ach_3_6</zorder>
  <zorder>__ach_3_7</zorder>
  <zorder>__ach_3_8</zorder>
  <zorder>__ach_3_9</zorder>
  <zorder>__ach_3_10</zorder>
  <zorder>__ach_3_11</zorder>
  <zorder>__ach_3_12</zorder>
  <zorder>__ach_2_1</zorder>
  <zorder>__ach_2_2</zorder>
  <zorder>__ach_2_3</zorder>
  <zorder>__ach_2_4</zorder>
  <zorder>__ach_2_5</zorder>
  <zorder>__ach_2_6</zorder>
  <zorder>__ach_2_7</zorder>
  <zorder>__ach_2_8</zorder>
  <zorder>__ach_2_9</zorder>
  <zorder>__ach_2_10</zorder>
  <zorder>__ach_2_11</zorder>
  <zorder>__ach_2_12</zorder>
  <zorder>__ach_1_1</zorder>
  <zorder>__ach_1_2</zorder>
  <zorder>__ach_1_3</zorder>
  <zorder>__ach_1_5</zorder>
  <zorder>__ach_1_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
